### ISSUE TITLE
Needs to double the users ram

### ISSUE THREAD

**Issue by yushaer** [TodePond/C#310]
In order for this language to be truly perfect it needs to be able to double the users ram otherwise it will be just like other mid languages

**Comment by yushaer** [comment]
This is how it can work `const const ram = ram *2`

**Comment by stohrendorf** [comment]
If you double infinite RAM, it will still be infinite, so yes.

**Comment by stitch** [comment]
Can i also just use Connectix RAM Doubler? Version 2.0.2 of course?

http://macintoshgarden.org/apps/ram-doubler-2

**Comment by everypizza1** [comment]
But what if we want more than 2 times the ram?

**Comment by Hri7566** [comment]
yes, there should probably be a parameter to concentrate exactly how much ram multiplication should happen

**Comment by ktlknss** [comment]
You could get infinite ram by doing

```java
tin infinity() => infinity() + 1!

var var ram = ram * infinity()
```
sorry if my _[insert current name of this programming language]_ isn't the best, i haven't used it in a while
